src/views/shared/portal/portal.styles.js

```diff
@@ -1,13 +1,14 @@
 import { createUseStyles } from 'react-jss'
 
-const usePortalStyles = createUseStyles({
+const usePortalStyles = createUseStyles((theme) => ({
   root: {
-    position: 'fixed',
+    position: 'absolute',
```

Comment:
This doesn't break other uses of Portal?

Reply:
I have the some doubt. Especially if this does affect to the `QR-Scanner` view

Reply:
I'll smote test the wallet again. But can you write specific cases to test?

Reply:
Yep! Basically when you are in the `transacion-form` view, you can enter the `to` address by scanning a QR. If the QR renders well it's all good. The same should work if you click the `My code` link in the header in the `home` view and then you click in the `Scan` button.

Reply:
Seems to me that it works fine, but it would be great if you could check as well @elias-garcia Just that we are 100% sure.

Reply:
@elias-garcia can we close this comment?

Reply:
yep, it's working fine! closing it